jenkins/automation/alex.groovy

```diff
@@ -1,7 +1,7 @@
 pipeline {
     agent {
           node {
-             label "ceph-build-node"
+             label "RE-Adhoc-VM"
```

Comment:
@shailesh-vaidya, this is not only for Alex work but serves ad-hoc tasks for RE.

Reply:
@subhalaxmisa, @gauravchaudhari02  - Can we conclude on this.

Reply:
@shailesh-vaidya No node named as `RE-Automated-Node` is available.

Reply:
@subhalaxmisa Can we rename the node name from `RE-Adhoc-VM` to `RE-Automated-Node`

Reply:
@shailesh-vaidya 
![image](https://user-images.githubusercontent.com/104340707/178651103-ab553584-b1bc-4e51-ac91-8b160aa4a918.png)

Reply:
@subhalaxmisa - Sorry If I am not clear about my statement. There is no node with the name `RE-Automated-Node` as of now. 
I feel we should use the `RE-Automation-Node` name for an agent which is currently named `RE-Adhoc-VM`. This will give more info about node usage.

Reply:
@shailesh-vaidya Got it and I have done the needful.
![image](https://user-images.githubusercontent.com/104340707/178653243-ba40e48d-c951-41b7-aa85-4795715a9be4.png)